app/assets/stylesheets/style.css

```diff
@@ -75,6 +75,22 @@ h6.dropdown-header {
   border: none;
 }
 
+
+.note-show h1 {
+  font-size: 2rem;
+}
+
+
+@media (max-width:400px) {
+  .note-show h1 {
+    font-size: 1.6rem;
+  }
+  .note-show h2 {
+    font-size: 1.4rem;
```

Comment:
This should be `1.8rem`. Thanks!!

Reply:
Hi @TildaDares, no worries. I can practice 😄